ccdproc/image_collection.py

```diff
@@ -112,6 +112,9 @@ def __init__(self, location=None, keywords=None, info_file=None,
         if keywords:
             self.keywords = keywords
 
+        if extension is not None:
```

Comment:
You mean the errors on an array being used as a bool?
I was wondering what to do about that. Thanks!